src/clusterfuzz/_internal/system/environment.py

```diff
@@ -862,7 +862,7 @@ def reset_current_memory_tool_options(redzone_size=0,
   bot_platform = platform()
 
   # Default options for memory debuggin tool used.
-  if tool_name in ['ASAN', 'HWASAN']:
+  if tool_name in ['ASAN', 'HWASAN', 'NOSANITIZER']:
```

Comment:
Could we just separate this out into its own branch and set `tool_options` to be `{}` instead?

Or alternatively, we just set `tool_options = {}` before line 865 and then we don't need the explicit branch.